### PR TITLE
Testsuite + other fixes

### PR DESCRIPTION
**Pull request by svillemot**

### CODE REVIEW

File: test/rtest-gdk-events.lisp [hunk 1]
```diff
@@ -119,7 +119,16 @@
...
                 (:TOUCH-CANCEL 40)
-                (:EVENT-LAST 41))
+                #+gdk-3-18 (:TOUCHPAD-SWIPE 41)
+                #+gdk-3-18 (:TOUCHPAD-PINCH 42)
+                #+gdk-3-22 (:PAD-BUTTON-PRESS 43)
+                #+gdk-3-22 (:PAD-BUTTON-RELEASE 44)
+                #+gdk-3-22 (:PAD-RING 45)
+                #+gdk-3-22 (:PAD-STRIP 46)
+                #+gdk-3-22 (:PAD-GROUP-MODE 47)
+                (:EVENT-LAST #-gdk-3-18 41
```
Comment (Ferada) [inline test/rtest-gdk-events.lisp]: Honestly I'd rather throw this out than update it for `:event-last` all the time.

File: test/rtest-gobject-signals.lisp [hunk 2]
```diff
@@ -3,6 +3,10 @@
 (def-suite gobject-signals :in gobject-suite)
 (in-suite gobject-signals)
 
+;; Ensure that GtkButton type is registered, otherwise there will be test
+;; failures at the first run
+(make-instance 'gtk-button)
```
Comment (Ferada) [inline test/rtest-gobject-signals.lisp:8]: Ah, no, we should do this differently, via `(glib::at-init () (foreign-funcall "xyz..._get_type" :int))` or even create a new function/macro for that so it doesn't get out of hand.

Maybe it's easiest if we ensure that `run-initializers` is called before running the tests.

Comment (svillemot) [reply test/rtest-gobject-signals.lisp:8]: I tried your solution, and it does not seem to work. I still get test failures, with several messages like the following on the console:

`(Program:20846): GLib-GObject-WARNING **: /build/glib2.0-B1uXKV/glib2.0-2.50.3/./gobject/gsignal.c:1253: unable to lookup signal "activate" of unloaded type 'GtkButton'`

Comment (Ferada) [reply test/rtest-gobject-signals.lisp:8]: I have trouble replicating this, could you post what you're running to get there?

So, from a clean image I run `(asdf:test-system '#:cl-cffi-gtk)` and while I get some errors, there's nothing in `*inferior-lisp*` that's similar.

Comment (svillemot) [reply test/rtest-gobject-signals.lisp:8]: I just run the whole testsuite (or just the tests in the `gobject-signals` sub-testsuite).

The important point is to start from a freshly restarted Lisp instance, where no `GtkButton` has ever been instantiated. And I use SBCL.

**Pull request merged**